cradle-core/src/main/java/com/exactpro/cradle/CradleStorage.java

```diff
@@ -65,6 +65,8 @@
 	protected abstract CompletableFuture<Void> doStoreMessageBatchAsync(StoredMessageBatch batch);
 	protected abstract void doStoreTimeMessage(StoredMessage message) throws IOException;
 	protected abstract CompletableFuture<Void> doStoreTimeMessageAsync(StoredMessage message);
+	protected abstract void doStoreStream(String streamName) throws IOException;
+	protected abstract CompletableFuture<Void> doStoreStreamAsync(String streamName);

```

Comment:
Nobody will call these methods. Streams of messages should be stored by CassandraCradleStorage as this is part of its data storing features.